tensorflow/python/estimator/inputs/pandas_io.py

```diff
@@ -50,7 +66,7 @@ def pandas_input_fn(x,
 
   Args:
     x: pandas `DataFrame` object.
-    y: pandas `Series` object. `None` if absent.
+    y: pandas `Series` object or `DataFrame`. `None` if absent.
```

Comment:
you probably need to update the target_column docstring also as it is not used when y is 'DataFrame'.

Reply:
Good point